react/Icons/Openapp.jsx

```diff
@@ -0,0 +1,16 @@
+// Automatically created, please run `scripts/generate-svgr-icon.sh /Users/vdnj/Documents/CozyCloud/cozy-ui/assets/icons/ui/openapp.svg` to regenerate;
```

Comment:
why is it different than in Album.jsx ?

`// Automatically created, please run `scripts/generate-svg-icon.sh assets/icons/ui/album.svg` to regenerate;`